hermes/Cargo.toml

```diff
@@ -77,3 +77,6 @@ bip32 = "0.5.1"
 ed25519-bip32 = "0.4.1"
 dashmap = "5.5.3"
 once_cell = "1.19.0"
+clap = "4.5.3"
+build-info = "0.0.36"
+derive_more =  { version= "0.99.17", default-features= false }
```

Comment:
```suggestion
derive_more = "0.99.17"
build-info-build = "0.0.36"
```

Reply:
don't we `default-features= false`?

Reply:
you should specify it for each crate directly, as you already done.
For workspace better to maintain just versions of dependencies, not how we are going to use them.

Reply:
Okey, thank for the clarification!